### DISCUSSION TITLE
Prevent external software to open known file extensions

### DISCUSSION THREAD

**Discussion by jpmaroco** [SafeExamBrowser/seb-win-refactoring#303] [Q&A]
Hi there,
Is there a way to prevent allowed external software to open certain file extensions? For example, can I allow Adobe Illustrator, but then prevent it from open .PDF files on the user's computer?
Thanks for any insights.

**Comment by dbuechel** [comment]
You should at least be able to prevent the download of such files, either by disabling downloads completely or then using a URL filter (see https://safeexambrowser.org/windows/win_usermanual_en.html#NetworkPaneFilterSection). However, once you allow the usage of third-party applications, you'd need to either find a way to suppress the functionality in the third-party application itself (SEB can't control that functionality, it only either allows or blocks an application) or then use another application which has a limited or more appropriate feature set.

**Comment by jpmaroco** [comment]
Thanks Damian.

Yes, I tought it needs to be a rescrition on the external application side….

Warm regards,

João
